Trivia Maze/src/application/Maze.java

```diff
@@ -186,20 +187,22 @@ public Room getRoom(int x, int y)
 
 	public void setDirection(Direction newDirection)
 	{
+		if(newDirection == null)
+			throw new IllegalArgumentException("Direction was null");
 		this.currentDirection = newDirection;
 	}
 
 	public void setExitX(int exitX)
 	{
-		if (exitX < 0 || exitX >= this.mazeColumns)
+		if (exitX < 0 || exitX >= 5)
 			throw new IllegalArgumentException("Passed X value is out of the bounds of the maze.");
 		
 		this.exitX = exitX;
 	}
 
 	public void setExitY(int exitY)
 	{
-		if (exitY < 0 || exitY >= this.mazeRows)
+		if (exitY < 0 || exitY >= 5)
```

Comment:
Should change from using 5 to using Main.ROWS

Reply:
Sure this program is not designed to scale.